src/main/java/nustracker/logic/parser/RemoveCommandParser.java

```diff
@@ -15,9 +17,17 @@
     @Override
     public RemoveCommand parse(String args) throws ParseException {
         requireNonNull(args);
-        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args,
-                PREFIX_STUDENTID,
-                PREFIX_EVENT);
+
+        ArgumentMultimap argMultimap = null;
```

Comment:
I think you should not assign anything here, like leave it unassigned. `ArgumentMultiMap argMultiMap`